templates/features.html

```diff
@@ -116,16 +122,9 @@
   }
 });
 
-search.addEventListener('keyup', function(e) {
-  // Clear results.
-  if (!e.target.value && e.keyCode != 27) { // ESC
-    featureList.filter();
-    chromeMetadata.selected = null;
-  } else {
-    //if (e.keyCode == 13) {  // Enter
-      featureList.filter(e.target.value);
-    //}
-  }
+search.addEventListener('input', function(e) {
+  featureList.filter(e.target.value);
```

Comment:
I believe it was deselecting the left hand side options. Why only when input is empty I'm not sure. But there are some problems with the previous logic anyway. Sometime when you start typing it just sets the left hand side to "No Active Development". I assume because it goes to else and filter function somewhat sets that.